src/Service/EntityManagerLookup.php

```diff
@@ -51,18 +51,17 @@ public function getManagerForEntity(string $entityClass): ManagerInterface
     {
         $reflect = new ReflectionClass($entityClass);
         $entityName = $reflect->getShortName();
-        $name = "App\\Entity\\Manager\\${entityName}Manager";
+        $name = sprintf('App\Repository\%sRepository', $entityName);
         if (!$this->container->has($name)) {
             throw new Exception(
-                sprintf('The manager for \'%s\' does not exist. Is the serivice public?', $name)
+                sprintf('The repository for \'%s\' does not exist. Is the service public?', $name)
             );
         }
 
         $manager = $this->container->get($name);
 
         if (!$manager instanceof ManagerInterface) {
-            $class = $manager->getClass();
-            throw new Exception("{$class} is not an Ilios Manager.");
+            throw new Exception("{$name} is not an Ilios Manager.");
```

Comment:
exception message should be `is not an Ilios Repository`.

Reply:
✔️